watchman/templates/watchman/dashboard.html

```diff
@@ -58,7 +58,14 @@ <h3 class="{% if overall_status %}text-success{% else %}text-danger{% endif %}">
                     {% trans "ERROR!" %}
 
                     <div class="btn-group btn-group-xs pull-right">
-                        <button type="button" class="btn btn-danger" data-toggle="modal" data-target="#{{ type_name|cut:' ' }}{% if status.name %}-{{ status.name|cut:' ' }}{% endif %}">{% trans "Traceback" %}</button>
+                        <button
+                            type="button"
+                            class="btn btn-danger"
+                            data-toggle="modal"
+                            data-target="#{{ type_name|cut:' ' }}{% if status.id %}-{{status.id}}{% elif status.name %}-{{ status.name|slugify|cut:' ' }}{% endif %}"
```

Comment:
Does it make sense to `slugify` the `status.id` as well? In case someone tries to put in some of the illegal special characters you mentioned (`.`, `:`, etc)?

Reply:
I think it's better to mention this restrictions in readme, is it ok?

Reply:
Yep, works for me!